### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by oliaboria**

### CODE REVIEW

File: app/src/routers/auth.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    '/jwt', 
+    (req, res) => {
+        let resp = null;
+        const user = authHelper.isAuthorized(req.body.email, req.body.password);
+
+        if (user) {
+            const token = jwt.sign(JSON.stringify(user), secret.key);
+
+            resp = {
+                code: 200,
```
Comment (vparashchanka) [inline app/src/routers/auth.js:20]: Why do we need code and message in the response body?

File: package.json [hunk 2]
```diff
@@ -34,6 +34,12 @@
   },
   "dependencies": {
     "express": "^4.16.2",
+    "jsonwebtoken": "^8.1.0",
+    "passport": "^0.4.0",
+    "passport-facebook": "^2.1.1",
```
Comment (vparashchanka) [inline package.json]: Please remove extra strategies.

**Review by scolt — APPROVED** [review]

File: app/src/routers/auth.js (hunk 1 above)
Comment (oliaboria) [reply app/src/routers/auth.js:20]: for the next homework))

File: package.json (hunk 2 above)
Comment (oliaboria) [reply package.json]: removed 💃

**Pull request merged**